dashboard/src/components/api/workspace/che-workspace.factory.ts

```diff
@@ -258,39 +283,34 @@ export class CheWorkspace {
   /**
    * Ask for loading the workspaces in asynchronous way
    * If there are no changes, it's not updated
-   * @returns {ng.IPromise<Array<che.IWorkspace>>}
    */
   fetchWorkspaces(): ng.IPromise<Array<che.IWorkspace>> {
-    let promise = this.remoteWorkspaceAPI.query().$promise;
-    let updatedPromise = promise.then((data: Array<che.IWorkspace>) => {
-      this.workspaces.length = 0;
-      this.workspacesById.clear();
-      // add workspace if not temporary
-      data.forEach((workspace: che.IWorkspace) => {
+    let promise = this.remoteWorkspaceAPI.query({'maxItems': 256}).$promise;
```

Comment:
should MAX_ITEMS be used here?

Reply:
Yes, MAX_ITEMS should be used here. Thanks.
I will fix it in another PR